geolocator_apple/lib/src/types/activity_type.dart

```diff
@@ -0,0 +1,21 @@
+/// Represents the possible iOS activity types
```

Comment:
nit: missing period.

```suggestion
/// Represents the possible iOS activity types.
```